dashboard/db/migrate/20160617163547_create_user_geos.rb

```diff
@@ -0,0 +1,16 @@
+class CreateUserGeos < ActiveRecord::Migration
+  def change
+    create_table :user_geos do |t|
+      t.references :user, index: true, unique: true, foreign_key: true, null: false
+      t.integer :user_id
+      t.timestamps null: false
+      t.datetime :indexed_at, null: false
+      t.string :ip_address
+      t.string :city
+      t.string :state
+      t.string :country
+      t.string :postal_code
+      t.string :latitude_longitude
```

Comment:
For consistency with existing code?
E.g., https://github.com/code-dot-org/code-dot-org/blob/staging/bin/cron/process_forms#L93, https://github.com/code-dot-org/code-dot-org/blob/staging/bin/cron/index_users_in_solr#L63, etc.

That said, I'm very open to splitting them apart, I'm just following existing usage. In terms of DB usage, if they are ever going to be used, we would definitely want them stored as decimals for distance computations (which will already be slow enough already).